api/nodes/serializers.py

```diff
@@ -72,8 +72,8 @@ class NodeSerializer(JSONAPISerializer):
     id = IDField(source='_id', read_only=True)
     type = TypeField()
 
-    category_choices = Node.CATEGORY_MAP.keys()
-    category_choices_string = ', '.join(["'{}'".format(choice) for choice in category_choices])
+    category_choices = Node.CATEGORY_MAP.items()
```

Comment:
Ohh, I didn't read the docs closely enough. OK, yeah, I think this is the right approach then. :+1: